packages/react/src/components/Tag/Tag-test.js

```diff
@@ -10,10 +10,42 @@ import Tag from '../Tag';
 import TagSkeleton from '../Tag/Tag.Skeleton';
 import { shallow } from 'enzyme';
 import { settings } from 'carbon-components';
+import { render, cleanup } from '@carbon/test-utils/react';
+import { afterEach } from 'jest-circus';
```

Comment:
```suggestion
```

Good ole VSCode most likely lol, `afterEach` should be a global from `jest` already 👍